experiment/simulation/index.html

```diff
@@ -1,13 +1,44 @@
 <!DOCTYPE html>
 <html>
-<head>
-	<!-- Add CSS at the head of HTML file -->
-    <link rel="stylesheet" href="./css/main.css">
-</head>
-<body>
-		<!-- Your code goes here-->
-
-	<!-- Add JS at the bottom of HTML file --> 
+    <head>
+        <meta charset = "utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/animejs/3.2.1/anime.min.js" integrity="sha512-z4OUqw38qNLpn1libAN9BsoDx6nbNFio5lA6CuTp9NlK83b89hgyCVq+N5FdBJptINztxn1Z3SaKSKUS5UP60Q==" crossorigin="anonymous"></script>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+        <link rel="stylesheet" href="./css/main.css">
+        <title>Preparation of Sols</title>
+    </head>
+    <body>
+        <div class="container" style=" display: flex; justify-content: center; align-items: center;">
+            <div class="grid">
+                <div class = "row">
+                    <div class = "message" id = "message" value="message"></div>
+                </div>
+                <div class = "row">
+                    <img id="bench" class = "bench" src="./images/bench.png" alt="Image">
+                    <div class = "water" id="water"></div>
+                    <img onclick="pour('top-beaker')" id="top-beaker" src="./images/beaker.png" alt="Image">
```

Comment:
You do not need to pass an argument to pour function here since you removed the parameter.

Reply:
Removed